src/pages/attribute-dictionary.js

```diff
@@ -246,7 +246,7 @@ const EventDefinition = memo(({ location, event, filteredAttribute }) => {
             margin-right: 0.5rem;
           `}
         >
-          Data {pluralize('source', event.dataSources.length)}
+          Data {pluralize('source:', event.dataSources.length)}
```

Comment:
Can you make sure that in the plural case it doesn't render as `source:s`? You may need to add the colon after the closing `}`

Reply:
oh I didn't realize there was a plural case. I think I fixed it,  but didn't see a way to select multiple sources in the navigation.